app/client/components/payment/update/ErrorSummary.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { Props } from "@guardian/src-helpers";
+import { css, SerializedStyles } from "@emotion/core";
+import { space } from "@guardian/src-foundations";
+import { size } from "@guardian/src-foundations/size";
+import { textSans } from "@guardian/src-foundations/typography";
+import { error as errorColors } from "@guardian/src-foundations";
+import { SvgAlertTriangle } from "@guardian/src-icons";
+
+interface SummaryProps extends Props {
+  /**
+   * The main message of the component
+   * e.g. the main error message, success message etc.
+   */
+  message: string;
+  /**
+   * Optional context information about the message
+   */
+  context?: React.ReactNode;
+}
+
+interface ErrorSummaryProps extends SummaryProps {
+  /**
+   * The error report link URL
+   */
+  errorReportUrl?: string;
+}
+
+const wrapperStyles = (color: string): SerializedStyles => css`
+  border: 4px solid ${color};
+  padding: ${space[1]}px;
+  display: flex;
+`;
+
+const iconStyles = (color: string): SerializedStyles => css`
+  display: flex;
+  flex: 0 1 auto;
+  margin-top: 1px;
+  svg {
+    fill: ${color};
+    height: ${size.xsmall}px;
+    width: ${size.xsmall}px;
+  }
+`;
+
+const messageStyles = (color: string, isBold = true): SerializedStyles => css`
+	${textSans.medium({ fontWeight: isBold ? "bold" : "regular" })}
+	color: ${color};
+`;
```

Comment:
looks like this could use prettier run on it ... pretty-quick should be working again in our pre commit hook in main, or you could just run prettier from your IDE

Reply:
hmmm, prettier isn't making changes